cmd/goal/account.go

```diff
@@ -563,103 +577,116 @@ func printAccountInfo(client libgoal.Client, address string, account v1.Account)
 	if len(createdAssets) == 0 {
 		fmt.Fprintln(report, "\t<none>")
 	}
-	for _, id := range createdAssets {
-		assetParams := account.AssetParams[id]
-
-		name := assetParams.AssetName
-		if len(name) == 0 {
-			name = "<unnamed>"
+	for _, createdAsset := range createdAssets {
+		name := "<unnamed>"
+		if createdAsset.Params.Name != nil {
+			_, name = unicodePrintable(*createdAsset.Params.Name)
```

Comment:
It may be unnecessary to pipe the string fields through `unicodePrintable` after #2555, but I've left them in just in case.